content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/tutorial-k8s.md

```diff
@@ -1,227 +1,388 @@
 ---
-date: '2021-09-21'
-title: "Deploying to a Local Kubernetes Cluster"
+date: '2021-11-14'
+title: "Deploying a Corda Cluster"
 menu:
   corda-5-alpha:
     parent: corda-5-alpha-tutorials-deploy
     identifier: corda-5-alpha-tutorial-deploy-k8s
     weight: 1000
 section_menu: corda-5-alpha
 ---
-This section describes how to build and deploy Corda to a Kubernetes cluster on your local development machine.
 
-## Create a Kubernetes Cluster
+This page describes how to deploy Corda 5 Alpha. It assumes all necessary [prerequisites](../prerequisites.html) have been installed.
 
-The following instructions assume that you have a single-node Kubernetes cluster running with a Docker daemon.
-Two options that meet these requirements and have been tested with these instructions are:
+## Download and Push Docker Images to a Registry
 
-* [Docker Desktop](#install-and-configure-docker-desktop)
-* [minikube](#install-minikube)
+The Corda Docker images must be in a Docker registry that is accessible from the Kubernetes cluster in which Corda will run. The images are provided in a `tar` file which can be loaded into a local Docker engine and then pushed from there to the registry.
 
-Docker Desktop provides a simpler user experience but commercial use in larger enterprises requires a paid subscription.
-See [Do I need to pay to use Docker Desktop?](https://docs.docker.com/desktop/faqs/general/#do-i-need-to-pay-to-use-docker-desktop) for more details.
+1. Download `corda-worker-images.tar` from the [R3 Customer Hub](https://r3.force.com/).
```

Comment:
looking at what @ronanbrowne created, `corda-worker-images.tar` will probably be suffixed w/ version, i.e. `corda-worker-images-<version>.tar`

Reply:
Changed to `corda-worker-images-eagle.tar`